spec/features/user_spec.rb

```diff
@@ -43,9 +73,18 @@
       expect(page).to have_content('User updated.')
     end
 
+    it 'Verifies users role can be edited' do
+      @users_page.change_users_role('John', 'Editor')
+      expect(page).to have_content('Changed roles.')
+    end
+
     it 'Verifies user can be deleted' do
       @users_page.delete_user('John')
       expect(page).to have_content('User deleted.')
```

Comment:
same as above, better to add verification that user is not displayed on the page